labcore/analysis/hvapps.py

```diff
@@ -148,12 +159,43 @@ def data_select(self):
     @pn.depends("_data_select_widget.value")
     def info_panel(self):
         path = self._data_select_widget.value
+        # Setup data preview panel
+        if path is not None:
+            abs = path.absolute()
+            # Add all/any images to a scrolling feed
+            images = []
+            for file in os.listdir(abs):
+                # check if the file ends with png
+                if (file.endswith(".png")):
+                    img = pn.pane.PNG(str(path) + "/" + file, sizing_mode = "fixed", width = self.image_feed_width)
+                    images.append(img)
+                    images.append( pn.Spacer(height = img.height))
+            self.data_images_feed.objects = images 
+            self.data_images_feed.width = self.image_feed_width + self.image_feed_scroll_width
+            # load datadict into dictionary/list
+            dd = self.load_data(str(abs) + "/data")
```

Comment:
Add another FIXME that specifies that this assumes the default data.ddh5 file in that directory, and that at some point it should be general